process/attendance.md

```diff
@@ -0,0 +1,27 @@
+# Attending a WebAssembly meeting
+
+Meetings are held in-person at the location determined in the meeting
+invitation. The invitation will contain logistics on location, time, address,
+parking, nearby hotels, Wi-Fi, and host contact. This invitation will be sent no
+later than 2 weeks before the meeting, through a pull request on the
+[meetings repository](https://github.com/WebAssembly/meetings) as well as
+through an email to the Community Group's mailing list.
+
+It is expected that a small breakfast, snacks, and beverages will be offered on
+premise, either for free or for a reasonable fee. The host may organize a dinner
+as well.
```

Comment:
May I suggest that if the host is organizing a dinner, people RSVP some number of weeks ahead of time. See https://github.com/tc39/Reflector/issues/65